llvm/tools/llvm-cov/CodeCoverage.cpp

```diff
@@ -1023,6 +1023,12 @@ int CodeCoverageTool::doShow(int argc, const char **argv,
   cl::alias ShowOutputDirectoryA("o", cl::desc("Alias for --output-dir"),
                                  cl::aliasopt(ShowOutputDirectory));
 
+  cl::opt<bool> BinaryCounters(
+      "binary-counters", cl::Optional,
+      cl::desc("Show when lines/branches are covered (1) or uncovered (0) "
```

Comment:
Show binary counters instead of integer execution counts?

Reply:
How about "Show binary counters (1/0) in lines and branches instead of..."?